ajax/libs/buy-button-js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "buy-button-js",
+  "filename": "buybutton.js",
+  "author": "Shopify Inc.",
+  "version": "0.4.13",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com:Shopify/buy-button-js.git"
+  },
+  "keywords":[
+    "ecommerce",
+    "buy-button-js"
+  ],
+  "autoupdate": {
```

Comment:
We should use npm auto-update here because the author only build and publish the files under `dist` on npm.
(Remove whole `autoupdate` which in Line14-Line25)
It would be like
```javascript
  "npmName": "@shopify/buy-button-js",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    }
  ],
```
For more details please see https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md.